assets/product-info.js

```diff
@@ -2,81 +2,324 @@ if (!customElements.get('product-info')) {
   customElements.define(
     'product-info',
     class ProductInfo extends HTMLElement {
+      quantityInput = undefined;
+      quantityForm = undefined;
+      onVariantChangeUnsubscriber = undefined;
+      cartUpdateUnsubscriber = undefined;
+      swapProductUtility = undefined;
+      abortController = undefined;
+
       constructor() {
         super();
-        this.input = this.querySelector('.quantity__input');
-        this.currentVariant = this.querySelector('.product-variant-id');
-        this.submitButton = this.querySelector('[type="submit"]');
```

Comment:
`currentVariant` can be grabbed from the product form class, `submitButton` wasn't used anywhere.